inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/perf_counters.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "multi-device/multi_device_config.hpp"
+#include "behavior/perf_counters.hpp"
+namespace {
+    const std::vector<std::map<std::string, std::string>> configs = {
+            {}
+    };
+
+    const std::vector<std::map<std::string, std::string>> Multiconfigs = {
```

Comment:
And what about other plugins?

Reply:
There were no instances in other plugins.